internal/common/db.go

```diff
@@ -105,6 +105,9 @@ type InfraEnv struct {
 
 	// Timestamp for expiration of the image
 	ImageExpiresAt strfmt.DateTime `json:"image_expires_at" gorm:"type:timestamp with time zone"`
+
+	// Hosts relationship
+	Hosts []*Host `json:"hosts" gorm:"foreignkey:InfraEnvID;association_foreignkey:ID"`
```

Comment:
Let's add a note, that in case somebody later would want to access Hosts in InfraEnv, he must be aware that it is not with eager loading. He can add it by himself, if he wants